src/app/courses/courses.component.html

```diff
@@ -1,119 +1,30 @@
-        <option *ngFor="let sub of subjectLevels" [value]="sub">{{sub}}</option>
-      </select>
-    </div>
-  </div>
-  <div class="row">
-    <div class="form-group col-4">
-      <label>Start Date</label>
-      <input type="date" class="form-control" formControlName="startDate">
-      <planet-form-error-messages [control]="courseForm.controls.startDate"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-4">
-      <label>End Date</label>
-      <input type="date" class="form-control" formControlName="endDate">
-      <planet-form-error-messages [control]="courseForm.controls.endDate"></planet-form-error-messages>
-    </div>
-    <div class="form-group radio col-4">
-      <div>Frequency</div>
-      <!-- radio buttons need same name to allow only one value to be selected! -->
-      <label>
-        <input type="radio" name="daysToggler" (change)="toogleWeekly(true)"> Daily
-      </label>
-      <label>
-        <input type="radio" name="daysToggler" (change)="toogleWeekly(false)"> Weekly
-      </label>
-    </div>
-  </div>
-  <div class="form-check form-check-inline" *ngIf="!showDaysCheckBox">
-    <div>Day</div>
-    <label class="form-check-label" *ngFor="let day of days">
-      <input class="form-check-input" type="checkbox" (change)="onDayChange(day, $event.target.checked)"> {{day}}
-    </label>
-  </div>
-  <div class="row">
-    <div class="form-group col-4">
-      <label>Start Time</label>
-      <input type="time" class="form-control" formControlName="startTime">
-      <planet-form-error-messages [control]="courseForm.controls.startTime"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-4">
-      <label>End Time</label>
-      <input type="time" class="form-control" formControlName="endTime">
-      <planet-form-error-messages [control]="courseForm.controls.endTime"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-4">
-      <label>Location</label>
-      <input type="text" class="form-control" formControlName="location">
-    </div>
-  </div>
-  <div class="row">
-    <div class="form-group col-2">
-      <label for="backgroundCol">Background Color</label>
-      <input type="color" class="form-control" formControlName="backgroundColor">
-      <planet-form-error-messages [control]="courseForm.controls.backgroundColor"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-2">
-      <label>Foreground Color</label>
-      <input type="color" class="form-control" formControlName="foregroundColor">
-      <planet-form-error-messages [control]="courseForm.controls.foregroundColor"></planet-form-error-messages>
-
-    </div>
-  </div>
-  <button type="submit" class="btn btn-primary">Submit</button>
-  <button type="button" class="btn btn-danger" (click)="cancel()">Cancel</button>
-</form>
-
-<p>Form value: {{ courseForm.value | json }}</p>
-<p>{{ courseForm.valid }}</p>
+<div class="container">
```

Comment:
We are using the `.container` class in a parent component. Som you don't need to use it here.